apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -351,6 +349,234 @@ paths:
+                      This parameter can be included in a request one or more times.
+
+                      Examples:
+                        - `service-date=gt2023-01-02&service-date=lt2023-05-01`
+                        - `service-date=gt2023-01-02`
+                        - `service-date=lt2023-05-01`
+                    example: "lt2023-05-01"
+                  startIndex:
+                    type: string
+                    description: |-
+                      When fetching a _Bundle Response_ using pagination, this URL parameter represents an offset
+                      (starting point) into the list of elements for the _Request_.
+                      It is optional and defaults to 1 if not supplied.
+                      A value 0 is not allowed and negative indices are not currently supported.
+
+                      Example:
+                      - `startIndex=100`
+                    example: 100
+                  type:
+                    type: string
+                    description: |-
+                      A list of one or more comma-separated claim types to be included in the request;
+                      within BFD, the claim types represent an _OR_ inclusion logic meaning any claims matching one of the specified
+                      claim types will be checked
+
+                      Supported Claim Type values:
+                        - `fiss`
+                        - `mcs`
+
+                      Examples:
+                        - `type=fiss,mcs`
+                        - `type=fiss`
+                    example: "fiss,mcs"
+          required: true
+        responses:
+          '200':
+            description: Success
+            content:
+              application/x-www-form-urlencoded:
+  /v2/fhir/ClaimResponse/_search:
+    post:
+      tags: [v2,ClaimResponse]
+      summary: 'search-type: Search for ClaimResponse instances'
+      description: This is a search type
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                mbi:
+                  type: string
+                  description: |-
+                    **NOTE: THIS IS A REQUIRED FIELD**
+                    
+                    Fetch _Beneficiary_ data using a FHIR _MBI_ identifier; an MBI
+                    represents the medicare benficiary ID, or as much of that
+                    defined by the [FHIR specification](https://terminology.hl7.org/NamingSystem-cmsMBI.html) as:
+                    
+                    ```
+                    A combination of upper case ASCII letters ('A'..'Z except for S, L, O, I, B, and Z.', numerals ('0'..'9'),
+                    '-', with a length limit of 11 characters.
+                    ```
+                    
+                    The following are all valid values for MBI, and all might
+                    represent the same resource:
```

Comment:
I'm not sure `all might represent the same resource` holds true here, since you removed the hashed MBI